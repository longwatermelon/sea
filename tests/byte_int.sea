fn main() -> int {
    let x:byte = 200b;
    let y:int = 100;

    let z:byte = x+y-50;
    return z;
}
