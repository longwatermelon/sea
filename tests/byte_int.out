RETURN_CODE:250
STDOUT:
